### ISSUE TITLE
Functional component as cell render flickering (React)

### ISSUE THREAD

**Issue by Luke93dev** [ag-grid/ag-grid#3967]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
 When trying to register React function component as cell renderers, I have noticed some flickering when scrolling grid in a fast way. It's not a problem when it's registered as class component.

Plunker: 
https://plnkr.co/edit/JjA4ERxztkVum1mx?preview

To reproduce this bug in Plunker, please resize "Row params" column to make whole its content visible. Then observe column values flickering after fast scroling or moving the vertical scroll.

I have found this bug described in this article: https://en.morzel.net/post/ag-grid-react-cell-renderers-performance

The same bug can be observed in this example of different renderers performance. Just select option Cell Render Type: React function component:
https://morzel85.github.io/blog-post-ag-grid-cell-renderer-speed/

**Expected behavior**
Functional components should have the same performance as class components.

Are there any plans to fix it soon? Do you know what is the newest ag-grid version that has a valid support for registering function components?

**Please tell us about your environment:**
ag-Grid version: 23.2.1
Browser: Chrome 73

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by kiril-matev** [comment]
Hello Luke!

Thank you for raising this with us.

We are aware of this issue, and we are already tracking it in our pipeline with the following reference and description:
**AG-4180 [React] React function component cell renderer briefly shows two copies of the cell value in a cell**

We're currently focused on our upcoming release and will be working on resolving this over the next 3 months.

I recommend checking the status of this item in our public product pipeline once a month. You'll be able to see when it's scheduled for implementation when it appears on the tab called NEXT RELEASE here:
https://www.ag-grid.com/ag-grid-pipeline/

For more information on when can you expect your ticket to be fixed please check:
https://ag-grid.zendesk.com/hc/en-us/articles/360002213531-Where-Is-My-Ticket-

Thanks again for bringing this up with us.

Regards,
Kiril

**Comment by wangJacker** [comment]
Has the problem been solved? same issue...

**Comment by kiril-matev** [comment]
Hello,

This issue was resolved in AG Grid v25.0 which was released in Jan 2021.

**Please note that we do not close update issues on github when they are fixed in the product. The best way to find out whether an issue you're tracking is fixed is to check the changelog page for the item you're looking for:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks for using AG Grid!
